.github/workflows/release.yml

```
@@ -461,9 +517,10 @@
             find ../provenance -name "*.intoto.jsonl" -type f -exec cp {} verification/multiple.intoto.jsonl \;
           fi
           
-          # Copy SBOM files
-          find ../sbom-artifacts -name "sbom-*.json" -type f -exec cp {} sbom/ \;
-          find ../sbom-artifacts -name "sbom-*.xml" -type f -exec cp {} sbom/ \;
+          # Copy SBOM files (both build-time and image-level)
+          find ../sbom-artifacts -name "sbom-build-*.json" -type f -exec cp {} sbom/ \;
+          find ../sbom-artifacts -name "sbom-build-*.xml" -type f -exec cp {} sbom/ \;
+          find ../sbom-artifacts -name "sbom-image-*.cyclonedx.json" -type f -exec cp {} sbom/ \;
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)